qt_ui/uncaughtexceptionhandler.py

```diff
@@ -33,7 +33,7 @@ def exception_hook(self, exc_type, exc_value, exc_traceback):
     def show_exception_box(self, message: str, exception: str) -> None:
         if QApplication.instance() is not None:
             QMessageBox().critical(
-                self.parent(),
+                QApplication.focusWidget(),
```

Comment:
Forces focus on exception windows so they don't stay hidden behind other windows...

Reply:
Extracted to its own PR...